corehq/apps/app_manager/templates/app_manager/summary_new.html

```diff
@@ -0,0 +1,63 @@
+{% extends "style/base_section.html" %}
+{% load xforms_extras %}
+{% load url_extras %}
+{% load hq_shared_tags %}
+{% load i18n %}
+{% load compress %}
+{% load djangular_tags %}
+
+{% block js %}
+    <script type="text/javascript" src="{% new_static 'app_manager/js/app_summary.ng.js' %}"></script>
+{% endblock js %}
+
+{% block js-inline %} {{ block.super }}
```

Comment:
`{% block uncompressed-js %}` will fix your build error. Alternatively, we could just do away with compressing `js-inline` altogether, and replace any references to `js-inline` with `js-uncompressed` (also flip the name around for `uncompressed-js` references). And just have two blocks, `js` that is clearly compressed and `js-uncompressed` that is clearly not. Do we really need to distinguish between inline and imported with compressor on anyway?